### PR TITLE
Change ROBOT tree -> gizmos tree browser

### PR DESCRIPTION
**Pull request by beckyjackson**
Note that I had to declare `owl:Thing` with a label in this PR in order for `gizmos.tree` to work properly - see related issue: https://github.com/ontodev/gizmos/issues/21

In order for this to run properly on DROID, `ontodev-gizmos` must be put into the DROID `requirements.txt`

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Good. I'd also like to be able to browse the upstream ontologies that we're importing from. At least a few of them, but ideally all of them.

**Comment by beckyjackson** [comment]
Right now, `gizmos.tree` doesn't support a `db` param in the query string, so I *think* we'd have to write different bash scripts for each ontology. One solution is to have a bash script "template" and just fill in the ontology database, or to put the `db` param in the query string (https://github.com/ontodev/gizmos/issues/21)

**Comment by jamesaoverton** [comment]
Ok, let's wait.

Since the user will be able to specify the `db` parameter, `tree.sh` will have to validate that user input. We should only allow a `db` that's in our `build/` directory (or wherever they are), not just any file on disk.

**Comment by beckyjackson** [comment]
Waiting on a release of `gizmos` containing these fixes: https://github.com/ontodev/gizmos/pull/23

**Comment by beckyjackson** [comment]
The tree view is implemented for ONTIE + imports. I want to remove the MIREOT script and change to `gizmos.extract`, but this depends on https://github.com/ontodev/gizmos/pull/27

Edit: The `-n` issue is fixed, but not released. Other than that, this PR is ready 🙂

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ resources/
 credentials.json
 .cogs
 .git-credentials
+.git-credentials
```
Comment (jamesaoverton) [inline .gitignore]: This is duplicate -- DROID probably added it by mistake.

**Review by jamesaoverton — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (beckyjackson) [reply .gitignore]: Fixed

**Pull request merged**